plugin/cache/README.md

```diff
@@ -103,3 +107,13 @@ Enable caching for all zones, keep a positive cache size of 5000 and a negative
     }
  }
  ~~~
+
+Enable caching for all zones but disable negative caching:
+
+~~~ corefile
+ . {
```

Comment:
We should get out of the habit of doing every example with `.`, please change this into into `example.org`, most examples also include a valid Corefile so add a forward in here or just *whoami*